front/public/locales/en/stdcm.json

```
@@ -114,6 +114,18 @@
   "stdcmErrors": {
     "bothPointAreScheduled": "The calculation cannot take into account both an origin and a destination schedule. You have to choose one or the other:",
     "infraNotLoaded": "Infrastructure is loading",
+    "missingFields": {
+      "departureTime": "The departure time",
+      "destination": "The destination",
+      "origin": "The origin",
+      "timeDestination": "the arrival time at the destination",
+      "timeOrigin": "the departure time at the origin",
+      "tractionEngine": "• The traction engine",
+      "totalMass": "• The total mass of the consist",
+      "totalLength": "• The total length of the consist",
```

Regarding previous comment
```suggestion
      "tractionEngine": "The traction engine",
      "totalMass": "The total mass of the consist",
      "totalLength": "The total length of the consist",
```

---

done